transport/TransportImpl.cpp

```diff
@@ -1691,10 +1694,19 @@ void TransportImpl::sendReports(uint64_t timestamp, bool rembReady)
             rembAdded = true;
         }
 
-        if (rtcpPacket->getLength() + MINIMUM_SR + std::min(receiverReportCount, 4u) * sizeof(rtp::ReportBlock) >
-            _config.mtu)
+        assert(rtcpPacket->getLength() <= packetLimit);
+
+        const auto spaceAvailable = packetLimit - rtcpPacket->getLength();
```

Comment:
We cannot use packetLimit as limit. Remove that from Packet.h and use _config.mtu instead. Otherwise the packet may grow to large to be sent and it is dropped.